src/style.css

```
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+form {
+  margin-top: 20px;
+}
+
+.search {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+.big_s {
```

Not the best option for class names. it should exactly describe the element and not use shortened words.
```suggestion
.big_search {
```